packages/message-manager/src/AbstractMessageManager.ts

```diff
@@ -32,7 +32,7 @@ export interface AbstractMessage {
   status: string;
   type: string;
   rawSig?: string;
-  securityProviderResponse?: Map<string, Json>;
+  securityProviderResponse?: Record<string, Json>;
```

Comment:
This seems like a breaking change. Maybe it would be better to extract this to another PR so that it's more noticeable?

Reply:
I have preview PRs ready in both the extension and client so can confirm this has been tested, this is purely to simplify the typing of the new controller as `BaseControllerV2` doesn't support `Map` in controller state.